.codebuild/azure-functional.yml

```diff
@@ -0,0 +1,46 @@
+version: 0.2
+env:
+  variables:
+    C7N_FUNCTIONAL: true
+    C7N_TEST_RUN: true
+  exported-variables:
+  - C7N_FUNCTIONAL
+  - C7N_TEST_RUN
+  secrets-manager:
+    AZURE_CLIENT_ID: "azure_root_testrunner:AZURE_CLIENT_ID"
+    AZURE_CLIENT_SECRET: "azure_root_testrunner:AZURE_CLIENT_SECRET"
+    AZURE_SUBSCRIPTION_ID: "azure_root_testrunner:AZURE_SUBSCRIPTION_ID"
+    AZURE_TENANT_ID: "azure_root_testrunner:AZURE_TENANT_ID"
+phases:
+  install:
+    runtime-versions:
+      python: 3.8
+    commands:
+    - python -m venv /venv
+    - . /venv/bin/activate
+    - python -m pip install --upgrade pip && pip install . && pip install -r requirements-dev.txt
+    - wget -q https://releases.hashicorp.com/terraform/0.13.3/terraform_0.13.3_linux_amd64.zip
```

Comment:
ideally terraform version should be an env variable so it can be overridden by a build invocation

Reply:
fixed